code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -79,3 +79,12 @@
 	if(dna.species.space_move(src))
 		return TRUE
 	return ..()
+
+/mob/living/carbon/human/movingcarefully()
+	if(buckled) //sitting down, being in a vehicle, etc
+		return FALSE 
+	if(stat) //iron out edgecases
+		return FALSE
+	if((cached_multiplicative_slowdown + dna.species.movement_delay(src)) >= CONFIG_GET(number/movedelay/walk_delay) && mobility_flags & MOBILITY_STAND) //later, i want walk to not slow you down past walk delay. later.
```

Comment:
Species that move slower shouldn't be counted as walking if they are on their species run intent. You should check if cached_multiplicative_slowdown > 0 (or whatever the default is) and leave the config out of it

Reply:
the intent here is not to limit to walking, but to nerf speed in combat. If a character is moving slow enough to consistently click on, it's fine.

anyways, reopen this pr, please.